aws_lambda_gateway/src/request.rs

```diff
@@ -31,6 +33,10 @@ struct ApiGatewayProxyRequestDef<'a> {
     headers: Option<DeserializeHeaders>,
     #[serde(default, borrow)]
     body: Option<Cow<'a, str>>,
+    #[serde(default, rename = "pathParameters")]
+    path_parameters:Option<HashMap<String, String> >,
```

Comment:
Could you feature flag these additions to the struct? I'd like to avoid having to deserialize into a hash map if the feature is not enabled.

Reply:
Added feature flags...the feature flags don't seem to work on the project level though.  I was able to compile within the aws_lambda_gateway module (cargo test --features="extensions") but not from the project (cargo test --features="gateway_extensions").